### PR TITLE
main-canvas added

### PR DESCRIPTION
**Pull request by ankitapuri**
I have made a PR for the main canvas :
- It has a table of rows and columns . 
- I have also added the feature to adjust the size but maximum permissible is 20

### CODE REVIEW

**Comment by BreadGenie** [comment]
> * I have also added the feature to adjust the size but maximum permissible is 20

Is there a reason it's capped to 20?

also you can use the project name as the heading

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+      <title>Pixel Art Maker</title>
```
Comment (BreadGenie) [inline index.html]: you can use the project name here in the ```<title>``` and the ```<h1>```

**Review by BreadGenie — APPROVED** [review]
everything looks great!
you can make small changes like the one I've commented on the h1 and title

**Comment by ankitapuri** [comment]
> > * I have also added the feature to adjust the size but maximum permissible is 20
> 
> Is there a reason it's capped to 20?
> 
> also you can use the project name as the heading

Yes If I exceed to 20 then it looks absurd on the single page. later I will create this issue to show it to the user on next page.

**Comment by ankitapuri** [comment]
> everything looks great!
> you can make small changes like the one I've commented on the h1 and title

I have made the required changes.

**Comment by anandbaburajan** [comment]
Looks good to me too! But a few tiny suggestions: the IDs associated with the elements in `index.html` can follow a common style. For example, stick to just snake_case maybe? Also the `package.json` description doesn't need to contain the gitter chatroom details. You can merge the PR after making the change! Thanks for contributing!

**Pull request merged**